draft-ietf-quic-transport.md

```diff
@@ -628,14 +664,17 @@ it creates a "cheat" where the client assumes a value.  That's a problem, so I'm
 tempted to suggest that this include any value less than 2^30 so that normal
 processing works - and can be properly exercised.]]
 
-A Retry packet is never explicitly acknowledged and is not re-transmitted.
+A Retry packet is never explicitly acknowledged in an ACK frame by a client.
+
+A server MUST only send a Retry in response to a client Initial packet.
 
-After receiving a Retry packet, the client immediately retransmits its handshake
-data in its original Initial packet, along with a copy of the Retry packet.
-\[\[TODO: replace this with Nick's proposal.]]
+If the Original Destination Connection ID field does not match the
+Destination Connection ID from the Initial packet it sent, it MUST
```

Comment:
maybe clarify here that `it` means the client?

Reply:
Dobe